prime-router/docs/proposals/0009-proposal-for-history-api.md

```diff
@@ -0,0 +1,232 @@
+    "reportId" : "18a8e461-b3f8-436d-b94e-4065c7b9d186",
+    "createdAt": "2021-06-30 13:03:00.305657-04",
+    "items": [
+        {
+            "trackingId": "reportIndex-0",
+            "destinationCount": 2,
+            "destinations": {
+                "COUNTY": [],
+                "STATE": [
+                    {
+                        "reportId": "33a8e478-b3d2-498d-b94e-4065f7b9e134",
+                        "receiverOrg": "md-doh",
+                        "recieverName": "elr",
+                        "format": "HL7_BATCH",
+                        "sentOn": "2021-06-29 11:00:00.086375-04"
+                    }
+                ],
+                "FEDERAL": [
+                    {
+                        "reportId": "44a8e478-c3f2-498d-b94d-5068f7b9e134",
+                        "receiverOrg": "hhsprotect",
+                        "recieverName": "elr",
+                        "format": "CSV",
+                        "sentOn": "2021-06-29 11:00:00.086375-04"
+                    }
+                ]
+            },
+            "trackingId": "reportIndex-2",
+            "destinationCount": 0,
+            "destinations": {
+                "COUNTY": [],
+                "STATE": [],
+                "FEDERAL": []
+            }
+        }
+    ]
+}
+```
+Alternatively, the jurisdiction breakdown can be removed and added as an element to each individual report. Items routing no where would have a destination count of zero and empty jurisdiction arrays (unless the structure is changed).
+
+Alternative Json Response:
+```json
+{
+    "result_set": {
+        "count": 20,
+        "offset": 0,
+        "limit": 20,
+        "total": 2
+    },
+    "reportId" : "18a8e461-b3f8-436d-b94e-4065c7b9d186",
+    "createdAt": "2021-06-30 13:03:00.305657-04",
+    "items": [
+        {
+            "trackingId": "reportIndex-0",
+            "destinationCount": 1,
+            "destinations": [
+                {
+                    "reportId": "33a8e478-b3d2-498d-b94e-4065f7b9e134",
+                    "receiverOrg": "md-doh",
+                    "recieverName": "elr",
+                    "jurisdiction": "STATE",
+                    "format": "HL7_BATCH",
+                    "sentOn": "2021-06-29 11:00:00.086375-04"
+                },
+                {
+                    "reportId": "44a8e478-c3f2-498d-b94d-5068f7b9e134",
+                    "receiverOrg": "hhsprotect",
+                    "recieverName": "elr",
+                    "jurisdiction": "FEDERAL",
+                    "format": "CSV",
+                    "sentOn": "2021-06-29 11:00:00.086375-04"
+                }
+            ]
+        },
+        {
+            "trackingId": "reportIndex-1",
+            "destinationCount": 0,
+            "destinations": []
+        }
+    ]
+}
+```
+The above Json only represents the sentOn status for the report. What are the different states/status of the destination reports that we want to represent? 
+* batch?
```

Comment:
So this is the key lineage question.  I feel like we can just provide:

- this Item entered the system at this time
- this Item went (or was supposed to go) to these outputs, along with info on whether it did or not.
- this Item has this metadata.